2024-03-13 20:43:35,286 INFO    StreamThr :2600 [internal.py:wandb_internal():86] W&B internal server running at pid: 2600, started at: 2024-03-13 20:43:35.285422
2024-03-13 20:43:35,287 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status
2024-03-13 20:43:35,292 INFO    WriterThread:2600 [datastore.py:open_for_write():85] open: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\run-3z2hj02h.wandb
2024-03-13 20:43:35,293 DEBUG   SenderThread:2600 [sender.py:send():380] send: header
2024-03-13 20:43:35,293 DEBUG   SenderThread:2600 [sender.py:send():380] send: run
2024-03-13 20:43:36,346 INFO    SenderThread:2600 [dir_watcher.py:__init__():211] watching files in: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files
2024-03-13 20:43:36,346 INFO    SenderThread:2600 [sender.py:_start_run_threads():1122] run started: 3z2hj02h with start time 1710333815.286423
2024-03-13 20:43:36,365 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: check_version
2024-03-13 20:43:36,366 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: check_version
2024-03-13 20:43:39,943 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: run_start
2024-03-13 20:43:39,985 DEBUG   HandlerThread:2600 [system_info.py:__init__():32] System info init
2024-03-13 20:43:39,986 DEBUG   HandlerThread:2600 [system_info.py:__init__():47] System info init done
2024-03-13 20:43:39,986 INFO    HandlerThread:2600 [system_monitor.py:start():181] Starting system monitor
2024-03-13 20:43:39,986 INFO    SystemMonitor:2600 [system_monitor.py:_start():145] Starting system asset monitoring threads
2024-03-13 20:43:39,986 INFO    HandlerThread:2600 [system_monitor.py:probe():201] Collecting system info
2024-03-13 20:43:39,991 INFO    SystemMonitor:2600 [interfaces.py:start():190] Started cpu monitoring
2024-03-13 20:43:39,992 INFO    SystemMonitor:2600 [interfaces.py:start():190] Started disk monitoring
2024-03-13 20:43:39,992 INFO    SystemMonitor:2600 [interfaces.py:start():190] Started gpu monitoring
2024-03-13 20:43:39,993 INFO    SystemMonitor:2600 [interfaces.py:start():190] Started memory monitoring
2024-03-13 20:43:40,020 INFO    SystemMonitor:2600 [interfaces.py:start():190] Started network monitoring
2024-03-13 20:43:40,029 DEBUG   HandlerThread:2600 [system_info.py:probe():196] Probing system
2024-03-13 20:43:40,030 DEBUG   HandlerThread:2600 [system_info.py:probe():244] Probing system done
2024-03-13 20:43:40,030 DEBUG   HandlerThread:2600 [system_monitor.py:probe():210] {'os': 'Windows-10-10.0.22000-SP0', 'python': '3.10.9', 'heartbeatAt': '2024-03-13T12:43:40.029178', 'startedAt': '2024-03-13T12:43:35.266418', 'docker': None, 'cuda': None, 'args': ('--task=P', '--weather=mix'), 'state': 'running', 'program': 'E:\\自动驾驶\\车位检测\\parking-space-Identification-in-different-weather-main\\train.py', 'codePathLocal': 'train.py', 'codePath': 'train.py', 'host': 'LAPTOP-QIVT4NMT', 'username': 'lenovo', 'executable': 'E:\\anaconda\\python.exe', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3201.0, 'min': 0.0, 'max': 3201.0}, 'cpu_freq_per_core': [{'current': 3201.0, 'min': 0.0, 'max': 3201.0}], 'disk': {'/': {'total': 465.74511337280273, 'used': 178.77838897705078}}, 'gpu': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'memory_total': 6442450944}], 'memory': {'total': 15.855525970458984}}
2024-03-13 20:43:40,030 INFO    HandlerThread:2600 [system_monitor.py:probe():211] Finished collecting system info
2024-03-13 20:43:40,030 INFO    HandlerThread:2600 [system_monitor.py:probe():214] Publishing system info
2024-03-13 20:43:40,030 DEBUG   HandlerThread:2600 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-03-13 20:43:40,031 DEBUG   HandlerThread:2600 [system_info.py:_save_pip():68] Saving pip packages done
2024-03-13 20:43:40,031 DEBUG   HandlerThread:2600 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-03-13 20:43:40,042 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:40,364 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\requirements.txt
2024-03-13 20:43:40,365 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\conda-environment.yaml
2024-03-13 20:43:42,081 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:44,125 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:46,170 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:47,158 DEBUG   HandlerThread:2600 [system_info.py:_save_conda():87] Saving conda packages done
2024-03-13 20:43:47,159 INFO    HandlerThread:2600 [system_monitor.py:probe():216] Finished publishing system info
2024-03-13 20:43:47,180 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:43:47,180 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: keepalive
2024-03-13 20:43:47,180 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:43:47,181 DEBUG   SenderThread:2600 [sender.py:send():380] send: files
2024-03-13 20:43:47,181 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2024-03-13 20:43:47,193 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:43:47,193 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:43:47,194 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:43:47,433 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\conda-environment.yaml
2024-03-13 20:43:47,433 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-metadata.json
2024-03-13 20:43:47,960 DEBUG   SenderThread:2600 [sender.py:send():380] send: telemetry
2024-03-13 20:43:48,205 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:48,439 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:43:50,236 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:50,459 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:43:50,964 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:43:52,269 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:54,309 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:54,464 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:43:54,466 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:43:54,466 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:43:54,467 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:43:54,486 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_created():271] file/dir created: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:43:56,351 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:56,354 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:43:56,354 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:43:56,355 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:43:56,355 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:43:56,356 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:43:56,357 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:43:56,357 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:43:56,357 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:43:56,358 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:43:56,494 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:43:56,494 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:43:58,384 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:43:58,525 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:00,423 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:00,491 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:00,492 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:00,492 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:00,493 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:00,533 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:01,138 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FF7280>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:44:01,138 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:44:01,139 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:44:01,508 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:02,207 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:44:02,207 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:44:02,208 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:44:02,405 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:02,405 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:02,460 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:02,540 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:02,540 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:02,542 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:02,542 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:02,542 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:02,543 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:02,543 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:02,544 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:03,546 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:04,500 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:04,548 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:06,536 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:06,565 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:06,693 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:06,832 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:06,832 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:06,833 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:07,578 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:07,579 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\config.yaml
2024-03-13 20:44:08,551 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:08,552 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:08,553 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:08,553 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:08,554 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:08,554 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:08,554 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:08,555 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:08,568 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:08,587 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:08,588 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:10,617 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:10,692 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:12,641 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:12,719 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:12,815 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:12,815 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:12,816 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:12,817 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:13,653 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:14,678 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:14,689 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:14,689 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:14,690 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:14,690 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:14,691 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:14,692 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:14,692 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:14,692 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:14,750 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:14,852 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD0FD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:44:14,852 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:44:14,852 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:44:15,686 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:16,724 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:16,773 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:17,213 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:44:17,229 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:44:17,229 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:44:18,571 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:18,793 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:18,872 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:18,872 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:18,872 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:18,874 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:19,772 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:20,787 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:20,810 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:20,811 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:20,812 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:20,812 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:20,813 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:20,813 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:20,813 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:20,814 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:20,828 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:21,796 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:22,824 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:22,862 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:23,832 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:24,890 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:24,941 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:24,942 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:24,942 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:24,943 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:25,874 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:26,907 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:26,907 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:26,908 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:26,909 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:26,910 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:26,910 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:26,910 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:26,910 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:26,910 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:26,911 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:26,924 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:27,918 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:28,938 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:28,939 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:28,956 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:29,600 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD3550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:44:29,600 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:44:29,601 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:44:29,601 INFO    wandb-upload_0:2600 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 95, in create_connection
    raise err
  File "E:\anaconda\lib\site-packages\urllib3\util\connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 714, in urlopen
    httplib_response = self._make_request(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 403, in _make_request
    self._validate_conn(conn)
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 1053, in _validate_conn
    conn.connect()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 363, in connect
    self.sock = conn = self._new_conn()
  File "E:\anaconda\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001EFE4FD3550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 487, in send
    resp = conn.urlopen(
  File "E:\anaconda\lib\site-packages\urllib3\connectionpool.py", line 798, in urlopen
    retries = retries.increment(
  File "E:\anaconda\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD3550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD3550>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\anaconda\lib\site-packages\wandb\sdk\lib\retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2580, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "E:\anaconda\lib\site-packages\wandb\sdk\internal\internal_api.py", line 2549, in upload_file
    response = self._upload_file_session.put(
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 647, in put
    return self.request("PUT", url, data=data, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 587, in request
    resp = self.send(prep, **send_kwargs)
  File "E:\anaconda\lib\site-packages\requests\sessions.py", line 701, in send
    r = adapter.send(request, **kwargs)
  File "E:\anaconda\lib\site-packages\requests\adapters.py", line 520, in send
    raise ConnectionError(e, request=request)
wandb.sdk.lib.retry.TransientError: None
2024-03-13 20:44:30,999 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:31,012 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:31,013 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:31,013 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:31,015 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:31,988 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:32,226 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:44:32,243 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:44:32,243 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:44:32,825 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:32,826 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:33,022 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:33,045 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:33,662 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:33,663 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:33,665 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:33,665 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:33,665 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:33,666 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:34,028 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:34,674 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:35,052 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:35,073 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:36,961 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:36,962 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:36,962 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:36,963 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:37,091 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:37,107 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:38,797 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:38,798 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:38,799 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:38,799 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:38,800 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:38,800 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:38,800 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:38,801 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:39,131 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:39,132 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:39,150 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:39,817 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:40,024 DEBUG   SystemMonitor:2600 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2024-03-13 20:44:40,025 DEBUG   SenderThread:2600 [sender.py:send():380] send: stats
2024-03-13 20:44:41,172 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:41,229 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:42,897 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:42,898 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:42,898 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:42,899 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:43,206 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:43,254 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:44,222 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:44,757 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:44,758 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:44,758 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:44,759 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:44,760 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:44,760 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:44,760 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:44,761 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:45,239 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:45,281 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:45,768 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:46,247 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:47,266 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:44:47,266 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:44:47,267 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:44:47,303 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:48,883 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:48,884 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:48,884 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:48,885 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:49,304 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:49,324 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:50,313 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:50,770 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:50,770 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:50,771 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:50,772 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:50,772 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:50,773 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:50,773 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:50,774 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:50,774 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:51,347 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:51,361 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:52,350 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:53,409 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:54,914 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:54,915 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:54,915 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:54,916 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:55,425 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:55,443 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:55,924 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:44:56,434 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:56,746 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:56,746 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:44:56,747 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:56,748 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:56,748 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:56,749 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:44:56,749 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:44:56,750 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:44:57,460 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:44:57,483 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:44:58,465 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:44:58,480 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FF7220>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:44:58,480 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:44:58,480 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:44:59,518 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:00,869 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:00,870 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:00,870 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:00,871 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:01,522 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:01,542 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:01,873 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:02,237 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:45:02,270 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:45:02,270 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:45:02,525 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:02,707 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:02,708 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:03,578 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:05,617 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:05,678 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:05,678 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:05,680 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:05,680 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:05,681 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:05,681 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:06,609 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:06,610 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:06,836 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:06,837 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:06,837 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:06,838 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:07,635 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:07,655 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:07,846 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:08,641 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:08,736 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:08,737 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:08,737 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:08,738 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:08,739 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:08,739 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:08,739 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:08,740 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:09,671 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:09,693 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:10,026 DEBUG   SenderThread:2600 [sender.py:send():380] send: stats
2024-03-13 20:45:10,681 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:11,775 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:12,852 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:12,852 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:12,853 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:12,853 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:12,854 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:13,747 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:13,802 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:14,694 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:14,695 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:14,695 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:14,696 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:14,697 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:14,697 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:14,697 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:14,698 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:14,759 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:14,760 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:15,831 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:16,800 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:17,246 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:45:17,278 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:45:17,279 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:45:17,856 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:18,713 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:18,841 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:18,842 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:18,842 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:18,842 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:19,499 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD0730>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:45:19,499 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:45:19,500 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:45:19,865 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:19,883 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:20,699 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:20,700 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:20,701 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:20,701 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:20,702 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:20,703 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:20,703 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:20,704 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:20,872 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:20,873 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:21,921 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:22,910 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:23,724 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:23,961 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:24,831 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:24,831 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:24,831 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:24,832 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:24,953 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:26,008 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:26,697 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:26,698 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:26,699 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:26,699 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:26,700 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:26,701 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:26,701 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:26,701 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:26,997 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:26,998 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:28,041 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:29,035 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:29,719 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:30,087 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:30,818 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:30,819 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:30,819 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:30,820 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:31,075 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:32,126 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:32,256 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:45:32,273 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:45:32,273 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:45:32,711 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:32,712 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:32,815 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:32,815 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:32,816 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:32,817 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:32,817 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:32,818 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:33,118 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:33,119 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:34,169 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:34,823 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:35,161 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:36,206 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:36,833 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:36,834 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:36,834 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:36,835 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:37,187 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:38,245 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:38,652 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:38,652 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:38,653 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:38,654 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:38,654 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:38,655 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:38,655 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:38,655 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:39,226 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:39,227 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:40,036 DEBUG   SenderThread:2600 [sender.py:send():380] send: stats
2024-03-13 20:45:40,036 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:40,273 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:41,264 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:42,367 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:42,789 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:42,790 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:42,790 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:42,791 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:43,318 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:44,401 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:44,644 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:44,645 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:44,646 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:44,646 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:44,647 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:44,647 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:44,648 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:44,648 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:45,364 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:45,365 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:45,651 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:46,434 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:47,266 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: internal_messages
2024-03-13 20:45:47,283 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: stop_status
2024-03-13 20:45:47,283 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: stop_status
2024-03-13 20:45:47,394 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:48,465 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:48,782 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:48,783 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:48,783 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:48,784 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:49,120 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/wandb-metadata.json?Expires=1710420230&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=KSl3Wf8zNrQeSk5KSzUo%2FHdTAc0o0sJw38aMDY50Fq6U7nUhcJM2uRaJLZ98iz7A%2BT1WeprD977jTCPc0m7gOpD9mTNUGBG%2BhqABW9S0E5qxpxnt2LNBLeUiG1WdgQIqhngWrgnfU6TtAxJr7JssomIr6ZKBkvqlUXmjN4AeT5FNy8hUQZJ3AJ4qqchmmuqLm5Pnn6PWSkkVwC8A6uz1eTLeLK%2BKExHpNtgGb7lyamFn4leRL8biAWn4LTIdksqvo7v87j3rRHnhIqYx3EQDJpBPoQEl%2BcnFkYuyvXUdfCZI1w9tSALOTfOJXzawD%2Bxxu4FzGQuGB5e5SRTJWo8%2F8A%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD27D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:45:49,120 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '1319'}
2024-03-13 20:45:49,120 ERROR   wandb-upload_0:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:45:49,431 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:50,483 ERROR   gpu       :2600 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-13 20:45:50,663 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:50,663 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:50,664 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: partial_history
2024-03-13 20:45:50,665 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:50,665 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:50,666 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:50,666 DEBUG   SenderThread:2600 [sender.py:send():380] send: history
2024-03-13 20:45:50,667 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: summary_record
2024-03-13 20:45:50,667 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:50,811 DEBUG   SenderThread:2600 [sender.py:send():380] send: exit
2024-03-13 20:45:50,811 INFO    SenderThread:2600 [sender.py:send_exit():585] handling exit code: 0
2024-03-13 20:45:50,811 INFO    SenderThread:2600 [sender.py:send_exit():587] handling runtime: 130
2024-03-13 20:45:50,812 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:50,812 INFO    SenderThread:2600 [sender.py:send_exit():593] send defer
2024-03-13 20:45:50,812 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,813 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 0
2024-03-13 20:45:50,813 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,813 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-13 20:45:50,813 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 1
2024-03-13 20:45:50,813 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,813 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 1
2024-03-13 20:45:50,813 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,813 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-13 20:45:50,813 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 2
2024-03-13 20:45:50,813 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,813 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 2
2024-03-13 20:45:50,814 INFO    HandlerThread:2600 [system_monitor.py:finish():190] Stopping system monitor
2024-03-13 20:45:50,821 DEBUG   SystemMonitor:2600 [system_monitor.py:_start():166] Finished system metrics aggregation loop
2024-03-13 20:45:50,821 INFO    HandlerThread:2600 [interfaces.py:finish():202] Joined cpu monitor
2024-03-13 20:45:50,821 DEBUG   SystemMonitor:2600 [system_monitor.py:_start():170] Publishing last batch of metrics
2024-03-13 20:45:50,821 INFO    HandlerThread:2600 [interfaces.py:finish():202] Joined disk monitor
2024-03-13 20:45:50,859 INFO    HandlerThread:2600 [interfaces.py:finish():202] Joined gpu monitor
2024-03-13 20:45:50,860 INFO    HandlerThread:2600 [interfaces.py:finish():202] Joined memory monitor
2024-03-13 20:45:50,860 INFO    HandlerThread:2600 [interfaces.py:finish():202] Joined network monitor
2024-03-13 20:45:50,861 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,861 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-13 20:45:50,861 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 3
2024-03-13 20:45:50,861 DEBUG   SenderThread:2600 [sender.py:send():380] send: stats
2024-03-13 20:45:50,861 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,861 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 3
2024-03-13 20:45:50,862 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,862 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-13 20:45:50,862 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 4
2024-03-13 20:45:50,862 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,862 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 4
2024-03-13 20:45:50,862 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,862 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-13 20:45:50,862 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 5
2024-03-13 20:45:50,862 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,862 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 5
2024-03-13 20:45:50,863 DEBUG   SenderThread:2600 [sender.py:send():380] send: summary
2024-03-13 20:45:50,864 INFO    SenderThread:2600 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2024-03-13 20:45:50,864 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,864 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-13 20:45:50,864 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 6
2024-03-13 20:45:50,864 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,864 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 6
2024-03-13 20:45:50,864 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,864 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-13 20:45:50,864 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 7
2024-03-13 20:45:50,864 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:45:50,864 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:50,866 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 7
2024-03-13 20:45:50,866 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:50,866 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-13 20:45:51,471 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:51,472 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json
2024-03-13 20:45:51,821 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:52,474 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 8
2024-03-13 20:45:52,475 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:52,475 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:52,475 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 8
2024-03-13 20:45:52,475 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:52,475 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-13 20:45:52,475 INFO    SenderThread:2600 [job_builder.py:build():281] Attempting to build job artifact
2024-03-13 20:45:52,476 INFO    Thread-12 :2600 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log
2024-03-13 20:45:52,476 INFO    SenderThread:2600 [job_builder.py:_get_source_type():404] no source found
2024-03-13 20:45:52,477 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 9
2024-03-13 20:45:52,477 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:52,477 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 9
2024-03-13 20:45:52,477 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:52,477 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-13 20:45:52,477 INFO    SenderThread:2600 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-13 20:45:52,822 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:53,488 INFO    SenderThread:2600 [dir_watcher.py:finish():388] scan: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files
2024-03-13 20:45:53,488 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\conda-environment.yaml conda-environment.yaml
2024-03-13 20:45:53,489 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\config.yaml config.yaml
2024-03-13 20:45:53,491 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\output.log output.log
2024-03-13 20:45:53,493 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\requirements.txt requirements.txt
2024-03-13 20:45:53,496 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-metadata.json wandb-metadata.json
2024-03-13 20:45:53,496 INFO    SenderThread:2600 [dir_watcher.py:finish():402] scan save: E:\自动驾驶\车位检测\parking-space-Identification-in-different-weather-main\wandb\run-20240313_204335-3z2hj02h\files\wandb-summary.json wandb-summary.json
2024-03-13 20:45:53,498 INFO    SenderThread:2600 [sender.py:transition_state():613] send defer: 10
2024-03-13 20:45:53,498 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:53,498 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: defer
2024-03-13 20:45:53,499 INFO    HandlerThread:2600 [handler.py:handle_request_defer():170] handle defer: 10
2024-03-13 20:45:53,500 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: defer
2024-03-13 20:45:53,500 INFO    SenderThread:2600 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-13 20:45:53,500 INFO    SenderThread:2600 [file_pusher.py:finish():175] shutting down file pusher
2024-03-13 20:45:53,830 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:53,830 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:54,835 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:54,835 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:55,835 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:55,836 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:56,845 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:56,845 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:57,854 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:57,855 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:58,859 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:58,859 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:45:59,874 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:45:59,874 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:00,876 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:00,876 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:46:00,876 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:01,885 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:01,885 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:02,885 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:02,886 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:03,890 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:03,891 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:04,898 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:04,898 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:05,912 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: status_report
2024-03-13 20:46:05,912 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:05,912 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:05,957 ERROR   wandb-upload_1:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/conda-environment.yaml?Expires=1710420356&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=j5kCncd%2BIW3pQ%2BZrIRDJ24rR%2B4yGvzjepMQbUgENq9WPfYdJg9KsDyBdmC8kvrXap5T5dORDw83yYGEguQE%2ByS2oKtC7FzHuJS3DxjGagMoXRJeKPHxAOLxkbRtgHYaEU2jx3U653rWQsweohqJ%2FVlrvo1TdLBETidI39CyaLyeGO1XiUKBtOg0tebb%2B8pIMCtBhK%2FKWuxbcN%2FVkZgnXCKe9Rx5JCKbseN9VpidDWD4bmv2ayr%2FbW7hQ1TFNQzwc4TfMlpKe7fU0qP277DBc3DyeyQpKtlFy1LQxXBe5oIO4HjbjcjVWimUidLL4bcuilpXZYcHvKHjsGIl8AgsGWw%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/conda-environment.yaml?Expires=1710420356&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=j5kCncd%2BIW3pQ%2BZrIRDJ24rR%2B4yGvzjepMQbUgENq9WPfYdJg9KsDyBdmC8kvrXap5T5dORDw83yYGEguQE%2ByS2oKtC7FzHuJS3DxjGagMoXRJeKPHxAOLxkbRtgHYaEU2jx3U653rWQsweohqJ%2FVlrvo1TdLBETidI39CyaLyeGO1XiUKBtOg0tebb%2B8pIMCtBhK%2FKWuxbcN%2FVkZgnXCKe9Rx5JCKbseN9VpidDWD4bmv2ayr%2FbW7hQ1TFNQzwc4TfMlpKe7fU0qP277DBc3DyeyQpKtlFy1LQxXBe5oIO4HjbjcjVWimUidLL4bcuilpXZYcHvKHjsGIl8AgsGWw%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD1B10>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:46:05,958 ERROR   wandb-upload_1:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '9492'}
2024-03-13 20:46:05,958 ERROR   wandb-upload_1:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:46:06,290 ERROR   wandb-upload_2:2600 [internal_api.py:upload_file():2556] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/config.yaml?Expires=1710420356&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ytn886oeHa9GCYMRTmzxFOdPX0J6u32t2xhafIYdwnlHClz7cqnboabl2Gli8nRX6HYMDqo2A1F2TiEWqWDb1mMj0fWi9ga6wzcucSyffPJJY0A8reM8Oy6lgvZ0wtnXlsfaWh5sgDuWk%2F0A5FnQkavKLs46utV87B%2BIKh81vILObFAc3G00HK%2FnDp4MyTHGJjpsNKIeuhCqVDdvy0wwteN4HrjyfwNfVrHoxO5xzfh6zZFoUuY7biJ%2FdL8Zm3zs%2FPEhVLrKYZaiX0AE1rNS2DbJ4%2BVdhM5OiEyFkv0jkvqPZFJRoKk5M6PGNTi4qt6WaWmYRHUWrw0pl%2FYL9umXbA%3D%3D: HTTPSConnectionPool(host='storage.googleapis.com', port=443): Max retries exceeded with url: /wandb-production.appspot.com/princeporch/uncategorized/3z2hj02h/config.yaml?Expires=1710420356&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=Ytn886oeHa9GCYMRTmzxFOdPX0J6u32t2xhafIYdwnlHClz7cqnboabl2Gli8nRX6HYMDqo2A1F2TiEWqWDb1mMj0fWi9ga6wzcucSyffPJJY0A8reM8Oy6lgvZ0wtnXlsfaWh5sgDuWk%2F0A5FnQkavKLs46utV87B%2BIKh81vILObFAc3G00HK%2FnDp4MyTHGJjpsNKIeuhCqVDdvy0wwteN4HrjyfwNfVrHoxO5xzfh6zZFoUuY7biJ%2FdL8Zm3zs%2FPEhVLrKYZaiX0AE1rNS2DbJ4%2BVdhM5OiEyFkv0jkvqPZFJRoKk5M6PGNTi4qt6WaWmYRHUWrw0pl%2FYL9umXbA%3D%3D (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001EFE4FD1060>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2024-03-13 20:46:06,290 ERROR   wandb-upload_2:2600 [internal_api.py:upload_file():2558] upload_file request headers: {'User-Agent': 'python-requests/2.29.0', 'Accept-Encoding': 'gzip, deflate, br', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '456'}
2024-03-13 20:46:06,290 ERROR   wandb-upload_2:2600 [internal_api.py:upload_file():2560] upload_file response body: 
2024-03-13 20:46:06,916 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:06,916 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:07,919 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:07,919 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
2024-03-13 20:46:08,921 DEBUG   HandlerThread:2600 [handler.py:handle_request():144] handle_request: poll_exit
2024-03-13 20:46:08,921 DEBUG   SenderThread:2600 [sender.py:send_request():407] send_request: poll_exit
